cuckoo_time_translator_python/python/cuckoo_time_translator_python/device_time_bags.py

```diff
@@ -4,9 +4,9 @@
 import rosbag
 import numpy as np
 
-from timestamp_series import TimestampSeries
-from timestamp_series import TimestampSeries
-from tools import *
+from .timestamp_series import TimestampSeries
+from .timestamp_series import TimestampSeries
+from .tools import *
```

Comment:
I think this breaks python2 (If I remember my struggles with . correctly)
If we care about that what should work in both versions is specifying the full path, i.e.
from cuckoo_time_translator_python.timestamp_series import TimestampSeries

(also one import is duplicated)

Reply:
Thanks. To keep things simple I've decided to drop Python2 support. The Python part should not be used much if at all out there and Python2 was faded out quite some time ago now.